docs/mmteb/readme.md

```diff
@@ -0,0 +1,64 @@
+
+### 🖥️ 2: Contribute a new task
+
+MTEB currently consists of 8 embedding tasks including tasks such as STS, retrieval, reranking, and more. If you feel like there is a category of tasks that is not yet covered, we would welcome contributions of these as well.
+
+### 🔍 3: Contribute new scores
+
+Once we have the datasets, we want to evaluate models on them. We welcome evaluation scores for models, which will be added to the leaderboard.
+
+### 🔓 4: Review PRs
+
+We welcome reviews of PRs adding new datasets. If you wish to review PRs of a specific language feel free to contact members of the MTEB team.
+
+## Authorship
+
+We follow a similar approach as in the [SeaCrowd Project](https://github.com/SEACrowd#contributing-to-seacrowd) and use a point-based system to determine co-authorships. 
+
+To be considered a co-author, at least 10 contribution points are required. The position of contributors in the author list is determined by the score they acquire, higher scores will appear first.
+
+To monitor how many points you have obtained, the contribution point tracking is now live at [this sheet](POINTS.md) and we recommend updating the score along with your PR. Past contributions also count. 
+
+Everyone with sufficient points will also be added to the MTEB GitHub and Huggingface repository as a contributor.
+
+# Contribution point guideline
+The contribution points are computed using the following table:
+
+> **Note**: The purpose of the point system is not to barrier collaboration, but to reward contributions. We might adjust the point requirement lower to accommodate more co-authorship if needed.
+
+
+| Contribution type   | Demand              | Points  | Description                                                                                                       |
+| ------------------- | ------------------- | ------- | ----------------------------------------------------------------------------------------------------------------- |
+| New dataset         | As many as possible | 2+bonus | The first dataset for a language x task gains 4 bonus points. |
+| New task            | If relevant         | 10      | Task 2.                                                                                                           |
+| Dataset annotations | On demand           | 1       | Adding missing dataset annotations to existing datasets.                                                          |
+| (bug)fixes            | On demand           | 1-10    | Points depends the effect of code changes. If you want to find issues related to the MMTEB you can find them [here](https://github.com/embeddings-benchmark/mteb/milestone/1). |
+| Running Models      | On demand           | 1       | Task 3.                                   |
+| Review PR           | On demand           | 2       | Task 4.                                   |
```

Comment:
So for new English datasets we also give 2 points, right?

Reply:
Yep!